thoth/common/helpers.py

```diff
@@ -196,3 +196,16 @@ def normalize_os_version(
 
     # Discard any minor release, if present.
     return os_version.split(".", maxsplit=1)[0]
+
+
+def get_solver_parts(solver_name: str) -> str:
```

Comment:
Oh thanks, I missed it! Should we move it to `.helpers`?

Reply:
It might be a good idea to keep it in the OpenShift class as it covers communication with OpenShift (and solvers are kept in OpenShift CM). Also, this method is used across components, so it might be a larger change in that case.